restli-server/src/main/java/com/linkedin/restli/server/config/RestLiMethodConfig.java

```diff
@@ -19,4 +18,9 @@
    * Gets whether query parameter validation against its parameter data template is enabled
    */
   boolean shouldValidateQueryParams();
+
+  /**
+   * Gets whether resource/path keys validation against its parameter data template is enabled
+   */
+  boolean shouldValidateResourceKeyParams();
```

Comment:
nit: change to `#shouldValidateResourceKeys` to be consistent with naming in `RestLiMethodConfigBuilder`, and also because these technically aren't really "params"

Also, you say `resource/path keys`, does that imply there's a difference between resource keys and path keys? I'm not really familiar with what a "resource key" is to be honest.

Reply:
Resource keys are identifiers for entity on which the resource takes action on.
there are two configs here one to validate keys in the path, 2nd in method level config which validates keys in batch requests or resource method argument/params